kedro-datasets/docs/source/conf.py

```diff
@@ -25,7 +25,7 @@
 
 # -- Project information -----------------------------------------------------
 
-project = "kedro-datasets"
+project = "kedro"
```

Comment:
Wondering if this should still be `kedro-datasets` (sorry I didn't catch that early on)

Reply:
Oops! Well, next time. At least it's now merged ! And https://docs.kedro.org/projects/kedro-datasets/en/latest/ is looking splendid.